package.json

```diff
@@ -14,7 +24,9 @@
     "devDependencies": {
         "@changesets/changelog-github": "^0.5.0",
         "@changesets/cli": "^2.27.1",
-        "glob": "^10.3.10"
+        "concurrently": "^8.2.2",
+        "glob": "^10.3.10",
+        "npm-watch": "^0.11.0"
```

Comment:
Could you prefer `chokidar`?

Reply:
switched to cookidar, i don't have strong feelings for or against any of them

Reply:
`npm-watch` uses `nodemon` which uses `chokidar` 😄